ui/png-icons/src/main/res/drawable/ic_widget_skip.xml

```diff
@@ -1,6 +1,6 @@
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
-    android:width="24dp"
-    android:height="24dp"
+    android:width="36dp"
+    android:height="36dp"
```

Comment:
The other widget icons use 48dp. I think it would make sense to use the same here

Reply:
Thanks, I have updated that file.